### PR TITLE
feat: configure exec facade to log based on the configuration

### PR DESCRIPTION
**Pull request by vmutafov**
Solution for: https://github.com/SAP/xsk/issues/452

### CODE REVIEW

File: modules/commons/commons-config/src/main/java/org/eclipse/dirigible/commons/config/Configuration.java [hunk 1]
```diff
@@ -448,7 +448,8 @@ public static void setSystemProperty(String key, String value) {
 			"DIRIGIBLE_PRODUCT_COMMIT_ID",
 			"DIRIGIBLE_PRODUCT_TYPE",
 			"DIRIGIBLE_INSTANCE_NAME",
-			"DIRIGIBLE_SPARK_CLIENT_URI"
+			"DIRIGIBLE_SPARK_CLIENT_URI",
+			"DIRIGIBLE_EXEC_DISABLE_COMMAND_LOGGING"
```
Comment (ThuF) [inline modules/commons/commons-config/src/main/java/org/eclipse/dirigible/commons/config/Configuration.java:452]: Let's align with the other environment variables:

`DIRIGIBLE_EXEC_DISABLE_COMMAND_LOGGING` -> `DIRIGIBLE_EXEC_COMMAND_LOGGING_ENABLED`

File: modules/engines/engine-command/src/main/java/org/eclipse/dirigible/engine/command/processor/CommandEngineExecutor.java [hunk 2]
```diff
@@ -221,6 +224,11 @@ public String executeCommandLine(String commandLine, Map<String, String> forAddi
 		return result;
 	}
 
+	private boolean shouldLogCommand() {
+		String shouldEnableLogging = Configuration.get("DIRIGIBLE_EXEC_DISABLE_COMMAND_LOGGING");
+		return Boolean.parseBoolean(shouldEnableLogging);
```
Comment (ThuF) [inline modules/engines/engine-command/src/main/java/org/eclipse/dirigible/engine/command/processor/CommandEngineExecutor.java:229]: You could add static import for the DIRIGIBLE_EXEC_COMMAND_LOGGING_ENABLED constant and update the block as follows:
```java
return Boolean.parseBoolean(Configuration.get(DIRIGIBLE_EXEC_COMMAND_LOGGING_ENABLED, "false"));
```

Comment (vmutafov) [reply modules/engines/engine-command/src/main/java/org/eclipse/dirigible/engine/command/processor/CommandEngineExecutor.java:229]: I prefer to keep it on two lines as it looks a bit more readable to me and there is no need for the default "false" as parseBoolean handles nulls as well.

As for if it should log or not log, I think it's better to not log by default

**Review by ThuF — APPROVED** [review]
+1

**Pull request merged**